content/release-10-15-0/announcements-10-15-0.md

```diff
@@ -31,6 +31,13 @@ The following API will no longer be supported:
 
 Instead, a time-to-live configuration or retention rules can be used to remove expired measurements data from the Operational Store.
 
+##### Breaking change in the Alarms/Events/Measurements API - unsupported query parameters will be rejected and required parameters will be introduced
+
+As of release 10.17+, it will no longer be possible to pass unsupported http query parameters, so also all request with wrongly typed parameters will be rejected.
```

Comment:
```suggestion
As of release 10.17+, it will no longer be possible to pass unsupported HTTP query parameters. This means that also all requests with wrongly typed parameters will be rejected.
```